packages/sdk/src/sdk/api/tracks/TracksApi.ts

```diff
@@ -228,7 +228,7 @@ export class TracksApi extends GeneratedTracksApi {
     }
 
     if (transcodePreview) {
-      if (!updatedMetadata.previewStartSeconds) {
+      if (updatedMetadata.previewStartSeconds === undefined) {
```

Comment:
Is it worth using `== null` here? Or does the input type prevent that?

Reply:
The input type is `number | undefined! so should be good here!